README.md

```diff
@@ -40,59 +43,103 @@ Ship is designed to provide first-time configuration UI and/or be used headless
 - Headless mode supports automated pipelines
 - Merge [Helm](https://helm.sh/) charts with override values and apply custom overlays with [Kustomize](https://kustomize.io/) to avoid merge conflicts when upstream or local values are changed
 - Watch upstream repos for updates & sync changes to your local version.
-- Deploy [Helm](https://helm.sh/) charts to a Kubernetes cluster without Tiller
+- Render [Helm](https://helm.sh/) charts to pure Kubernetes yaml that can be deployed to clusters without Tiller
 - Enables [GitOps](https://www.weave.works/blog/the-gitops-pipeline) workflows to update third party applications
 - Configuration workflow `ship.yaml` files can be included in [Helm](https://helm.sh/) chart repos, to customize the initial `ship init` experience
 
-# Installation
-There are two ways you can get started with Ship:
+# Getting Started
+
+## Installation
 
-## Installing locally
 Ship is packaged as a single binary, and Linux and MacOS versions are distributed:
 - To download the latest Linux build, run:
 ```shell
-curl -sSL https://github.com/replicatedhq/ship/releases/download/v0.28.0/ship_0.28.0_linux_amd64.tar.gz | tar xv && sudo mv ship /usr/local/bin
+curl -sSL https://github.com/replicatedhq/ship/releases/download/v0.29.0/ship_0.29.0_linux_amd64.tar.gz | tar xv && sudo mv ship /usr/local/bin
 ```
 
-- To download the latest MacOS build, run:
+- To download the latest MacOS build, you can either run:
 ```shell
-curl -sSL https://github.com/replicatedhq/ship/releases/download/v0.28.0/ship_0.28.0_darwin_amd64.tar.gz | tar xv && sudo mv ship /usr/local/bin
+curl -sSL https://github.com/replicatedhq/ship/releases/download/v0.29.0/ship_0.29.0_darwin_amd64.tar.gz | tar xv && sudo mv ship /usr/local/bin
 ```
 
-Ship is also available through [Homebrew](https://brew.sh/):
+- ... or you can install with [Homebrew](https://brew.sh/):
 ```shell
 brew tap replicatedhq/ship
 brew install ship
 ```
 
-After ship is installed, run it with:
+Alternately, you can run Ship in Docker, but you will need to mount and configure a shared volume, in order to persist any changes made within the Ship admin console when launched via Docker.
```

Comment:
I think rather than (or in addition to) an explanation about a shared volume being required, we should just include the `-v $PWD:/out` in the example docker run command.

Reply:
Yes indeed.  Examples updated